acoustid/future/api/handlers/monitoring.py

```
@@ -0,0 +1,13 @@
+import msgspec
+from starlette.requests import Request
+from starlette.responses import Response
+
+from ..utils import MsgspecResponse
+
+
+class Health(msgspec.Struct):
+    ready: bool
+
+
+def handle_health(request: Request) -> Response:
```

Ruff (0.8.2)

Unused function argument: `request`

(ARG001)